app/views/internal/feedback_messages/_abuse_reports.html.erb

```diff
@@ -44,26 +44,61 @@
             <span>
               <% if params[:status] == "Open" || params[:status].blank? %>
                 <% if reaction.reactable_type == "User" %>
-                  <%= form_for [:internal, reaction], html: { class: "d-inline" } do |f| %>
-                    <%= f.hidden_field :status, value: "confirmed" %>
-                    <%= f.submit "CONFIRMED", class: "btn btn-success btn-sm", data: { confirm: "Are you sure?" } %>
-                  <% end %>
+                  <button class="btn btn-success btn-sm js-reaction-confirm" type="button" data-reactable="user" data-id="<%= reaction.id %>" data-status="confirmed">CONFIRM</button>
```

Comment:
since this line is quite long, maybe we can split it like this:

```erb
<button
  class="btn btn-success btn-sm js-reaction-confirm"
  type="button"
  data-reactable="user"
  data-id="<%= reaction.id %>"
  data-status="confirmed">
  CONFIRM
</button>
```

Reply:
you know, I had done this, but it kinda looked weird to me 😆